indico/modules/events/surveys/util.py

```diff
@@ -38,3 +38,17 @@ def make_survey_form(questions):
             continue
         setattr(form_class, name, field_impl.get_wtforms_field())
     return form_class
+
+
+def create_preview_form(questions):
+    """Creates a WTForm from survey questions
+
+    This form is intended to be used as a preview form of a survey
+
+    :return An `IndicoForm` subclass
+    """
+    form_class = type(b'SurveySubmissionForm', (IndicoForm,), {})
+    for question in questions:
+        setattr(form_class, question.title, question.field.wtf_field)
```

Comment:
I'd use the same logic as in `make_survey_form` here: `name = 'question_{}'.format(question.id)`